bitnami/thanos/README.md

```diff
@@ -768,6 +775,7 @@ querier.strategyType                          -> query.strategyType
 querier.affinity                              -> query.affinity
 querier.nodeSelector                          -> query.nodeSelector
 querier.tolerations                           -> query.tolerations
+querier.podLabels                             -> query.podLabels
```

Comment:
I don't think this is is needed, right? The old `querier.podLabels` didn't exist.